cmd/kube-proxy/app/server_unsupported.go

```diff
@@ -0,0 +1,39 @@
+// +build !linux,!windows
+
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package app
+
+import (
+	"fmt"
+
+	"k8s.io/kubernetes/cmd/kube-proxy/app/options"
+)
+
+// NewProxyServerDefault creates a new ProxyServer object with default parameters.
+func NewProxyServerDefault(config *options.ProxyServerConfig) (*ProxyServer, error) {
+	return nil, fmt.Errorf("Proxying is not supported in this build")
+}
+
+// Tune conntrack, if requested
+func (s *ProxyServer) tuneConnTracker() error {
```

Comment:
conntrack is something we can't do in Windows, yet. We're evaluating new solutions like OpenVSwitch to deliver this and much more, e.g. stateful firewalling (flows). We're just trying to _interface_ it as much as we can right now so it becomes easier in the future to implement for Windows.